docs/griptape-framework/drivers/src/prompt_drivers_structured_output_multi.py

```diff
@@ -0,0 +1,28 @@
+import schema
+from rich.pretty import pprint
+
+from griptape.drivers import OpenAiChatPromptDriver
+from griptape.rules import JsonSchemaRule
+from griptape.structures import Pipeline
+from griptape.tasks import PromptTask
+
+pipeline = Pipeline(
+    tasks=[
+        PromptTask(
+            prompt_driver=OpenAiChatPromptDriver(
+                model="gpt-4o",
+                use_native_structured_output=True,
+                native_structured_output_strategy="tool",
+            ),
+            rules=[
+                JsonSchemaRule(schema.Schema({"color": "red"})),
+                JsonSchemaRule(schema.Schema({"color": "blue"})),
+            ],
```

Comment:
This is really hard for me to imagine what this would do (let alone decide to use it). How do the rules interact with the `tool` native output strategy?